openwisp_radius/migrations/0001_initial_freeradius.py

```diff
@@ -32,21 +18,19 @@ class Migration(migrations.Migration):
                 ('name', models.CharField(db_column='nasname', db_index=True, help_text='NAS Name (or IP address)', max_length=128, verbose_name='name')),
                 ('short_name', models.CharField(db_column='shortname', max_length=32, verbose_name='short name')),
                 ('type', models.CharField(default='other', max_length=30, verbose_name='type')),
-                ('ports', models.PositiveIntegerField(blank=True, null=True, verbose_name='ports')),
                 ('secret', models.CharField(help_text='Shared Secret', max_length=60, verbose_name='secret')),
-                ('server', models.CharField(blank=True, max_length=64, null=True, verbose_name='server')),
+                ('ports', models.PositiveIntegerField(blank=True, null=True, verbose_name='ports')),
                 ('community', models.CharField(blank=True, max_length=50, null=True, verbose_name='community')),
                 ('description', models.CharField(blank=True, max_length=200, null=True, verbose_name='description')),
-                ('organization', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='openwisp_users.Organization', verbose_name='organization')),
+                ('server', models.CharField(blank=True, max_length=64, null=True, verbose_name='server')),
             ],
             options={
+                'swappable': 'OPENWISP_RADIUS_NAS_MODEL',
+                'db_table': 'nas',
                 'verbose_name_plural': 'NAS',
                 'abstract': False,
                 'verbose_name': 'NAS',
-                'swappable': 'OPENWISP_RADIUS_NAS_MODEL',
-                'db_table': 'nas',
             },
-            bases=(openwisp_users.mixins.ValidateOrgMixin, models.Model),
```

Comment:
**Note:** `bases` is missing in all the classes and `makemigrations` didn't add it in the next file either! Should i add them manually? :blush: